docs/about-mono/releases/5.10.0.md

```diff
@@ -2,44 +2,161 @@
 layout: releasenotes
 title: 'Mono 5.10.0 Release Notes'
 version: 5.10.0
+releasedate: '2018-02-26'
 ---
 
 Release History
 ===============
 
-TODO
+* 5.10.0.141 - Preview Release _(27 Feb 2018)_
+* 5.10.0.140 - Stable Release _(26 Feb 2018)_
+* 5.10.0.140 - Preview Release _(24 Feb 2018)_
+* 5.10.0.137 - Preview Release _(23 Feb 2018)_
+* 5.10.0.132 - Preview Release _(22 Feb 2018)_
+* 5.10.0.129 - Preview Release _(19 Feb 2018)_
+* 5.10.0.125 - Preview Release _(19 Feb 2018)_
+* 5.10.0.116 - Preview Release _(15 Feb 2018)_
+* 5.10.0.107 - Preview Release _(14 Feb 2018)_
+* 5.10.0.105 - Preview Release _(13 Feb 2018)_
+* 5.10.0.103 - Preview Release _(10 Feb 2018)_
+* 5.10.0.101 - Preview Release _(9 Feb 2018)_
+* 5.10.0.97 - Preview Release _(8 Feb 2018)_
+* 5.10.0.94 - Preview Release _(7 Feb 2018)_
+* 5.10.0.91 - Preview Release _(6 Feb 2018)_
+* 5.10.0.83 - Preview Release _(5 Feb 2018)_
+* 5.10.0.79 - Preview Release _(2 Feb 2018)_
+* 5.10.0.78 - Preview Release _(31 Jan 2018)_
+* 5.10.0.71 - Preview Release _(30 Jan 2018)_
 
 Highlights
 ==========
 
-TODO
+* [Interpreter compiled by default](#interpreter)
+* [Default Interface Method support](#default-interface-method)
+* [Mono SDKs](#mono-sdks)
 
 In Depth
 ========
 
 Runtime
 -------
 
-TODO
+### Interpreter
+
+It is now compiled in by default, and you can use it by running `mono --interpreter`.
```

Comment:
maybe link to the interpreter blog post: http://www.mono-project.com/news/2017/11/13/mono-interpreter/